lib/find-python.js

```diff
@@ -0,0 +1,316 @@
+'use strict'
+
+const path = require('path')
+const log = require('npmlog')
+const semver = require('semver')
+const cp = require('child_process')
+const extend = require('util')._extend // eslint-disable-line
+const win = process.platform === 'win32'
+const logWithPrefix = require('./util').logWithPrefix
+
+function PythonFinder (configPython, callback) {
+  this.callback = callback
+  this.configPython = configPython
+  this.errorLog = []
+}
+
+PythonFinder.prototype = {
+  log: logWithPrefix(log, 'find Python'),
+  argsExecutable: [ '-c', 'import sys; print(sys.executable);' ],
+  argsVersion: [ '-c', 'import sys; print("%s.%s.%s" % sys.version_info[:3]);' ],
+  semverRange: process.env.NODE_GYP_PYTHON3 ? '>=2.7.0' : '>=2.6.0 <3.0.0',
```

Comment:
Please remove v2.6.0 to align with this project's README.md.

Reply:
This is only to extend the range of acceptable versions, not select which version to use.